text/src/autogluon/text/automm/constants.py

```diff
@@ -100,3 +114,7 @@
 CATEGORICAL_TRANSFORMER = "categorical_transformer"
 FUSION_MLP = "fusion_mlp"
 FUSION_TRANSFORMER = "fusion_transformer"
+
+# checkpoints
```

Comment:
Registering the default checkpoint names is good. We can also define `LAST_CHECKPOINT='last.ckpt'` and `MODEL_CHECKPOINT= 'model.ckpt'` here.